src/site/content/en/lighthouse-performance/speed-index/index.md

```diff
@@ -29,17 +29,14 @@ to generate the Speed Index score.
 
 {% Aside %}
 Speedline is based on the same principles as the
-[original speed index introduced by WebpageTest.org](https://github.com/WPO-Foundation/webpagetest-docs/blob/master/user/Metrics/SpeedIndex.md),
-but it computes the visual progression between frames using the
-[structural similarity (SSIM) index](https://en.wikipedia.org/wiki/Structural_similarity)
```

Comment:
Lighthouse moved off SSIM in https://github.com/googlechrome/lighthouse/pull/4980